common/config.py

```diff
@@ -0,0 +1,68 @@
+from configparser import ConfigParser
+from dataclasses import dataclass
+from typing import TypeVar
+from io import BytesIO
+from django.utils.encoding import force_text
+
+
+@dataclass(frozen=True)
+class Credentials:
+    user: str
+    password: str
+    email: str
+
+
+class CredentialsLoader:
+    """Loads credentials from config file using custom credentials and loader classes"""
+
+    def __init__(
+        self,
+        config_file,
+        credentials_class: TypeVar("Credentials") = None,
+        loader_class=None,
+    ):
+        """Initialises custom credentials and loader classes."""
+        self.credentials_class = credentials_class
+        self.loader_class = loader_class(config_file)
+
+    def read_config(self, verification_status: str) -> dict:
+        return self.loader_class.read_config(verification_status)
+
+
+class ConfigByteLoader:
+    """Loads data from byte stream input"""
+
+    def __init__(self, config_file: str):
+        """Reads byte stream input as binary file."""
+        with open(config_file, "rb") as f:
```

Comment:
suggest to use IO input() -> StringIO() ->  open(io_file) as f and use this code snippet in a separate PR that focuses on testing and implementation of ConfigByteLoader